### DISCUSSION TITLE
gatsby-plugin-netlify-cms and cms-config-url

### DISCUSSION THREAD

**Discussion by kalebheitzman** [gatsbyjs/gatsby#31709] [Help]
Hi all,

I'm including netlify-cms as a part of a theme and I want to use both manual init as well as specify a config.yml in the theme. It looks like my site is looking for config.yml in the main site instead of the theme however so it comes up as missing. The config.yml isn't being published.

With Netlify CMS you can specify a [cms-config-url](https://www.netlifycms.org/docs/configuration-options/#configuration-file) link tag to tell Netlify CMS where to look for your config file. Is there a way to do this from the plugin options?

Am I going about this all wrong. The goal is to ship a theme where content lives in the main site repository and the theme is separate from the content.

**Comment by wardpeet** [comment]
cc @erezrokah

**Reply by erezrokah** [reply]
Hi @wardpeet, thanks for tagging me. You can't configure the `cms-config-url` at the moment via the plugin.
It should be fairly easy to add it here:
https://github.com/gatsbyjs/gatsby/blob/2dde956bd1de6ad404416ec9dc4406a25951fe4b/packages/gatsby-plugin-netlify-cms/src/gatsby-node.js#L89
then pass it here https://github.com/gatsbyjs/gatsby/blob/2dde956bd1de6ad404416ec9dc4406a25951fe4b/packages/gatsby-plugin-netlify-cms/src/gatsby-node.js#L187

Maybe generalize it as `HtmlWebpackPlugin` options to pass down.

**_However_**, a simpler way might be to use [`manualInit`](https://www.gatsbyjs.com/plugins/gatsby-plugin-netlify-cms/#manualinit) with a JS based configuration, and even pass `load_config_file: false` to disable loading the `config.yml` completely. See [here](https://www.netlifycms.org/docs/beta-features/#manual-initialization) for more details
